pkg/k8shandler/elasticsearch.go

```diff
@@ -322,7 +351,44 @@ func GetMinMasterNodes(clusterName, namespace string, client client.Client) (int
 	return masterCount, payload.Error
 }
 
-func GetClusterHealth(clusterName, namespace string, client client.Client) (string, error) {
+func GetClusterHealth(clusterName, namespace string, client client.Client) (api.ClusterHealth, error) {
+
+	clusterHealth := api.ClusterHealth{}
+
+	payload := &esCurlStruct{
+		Method: http.MethodGet,
+		URI:    "_cluster/health",
+	}
+
+	curlESService(clusterName, namespace, payload, client)
+
+	/*
+	   type ClusterHealth struct {
+	   	Status						string	`json:"status"`
+	   	NumNodes					int32		`json:"numNodes"`
+	   	NumDataNodes			int32		`json:"numDataNodes"`
+	   	ActivePrimShards	int32		`json:"activePrimShards"`
+	   	ActiveShards			int32		`json:"activeShards"`
+	   	RelocatingShards	int32		`json:"relocatingShards"`
+	   	InitializingShards	int32	`json:"initializingShards"`
+	   	UnassignedShards	int32		`json:"unassignedShards"`
+	   }
+	*/
+
+	clusterHealth.Status = parseString("status", payload.ResponseBody)
+	clusterHealth.NumNodes = parseInt32("number_of_nodes", payload.ResponseBody)
+	clusterHealth.NumDataNodes = parseInt32("number_of_data_nodes", payload.ResponseBody)
+	clusterHealth.ActivePrimShards = parseInt32("active_primary_shards", payload.ResponseBody)
+	clusterHealth.ActiveShards = parseInt32("active_shards", payload.ResponseBody)
+	clusterHealth.RelocatingShards = parseInt32("relocating_shards", payload.ResponseBody)
+	clusterHealth.InitializingShards = parseInt32("initializing_shards", payload.ResponseBody)
+	clusterHealth.UnassignedShards = parseInt32("unassigned_shards", payload.ResponseBody)
+	//clusterHealth.PendingTasks = parseInt32("number_of_pending_tasks", payload.ResponseBody)
```

Comment:
What's this doing here?

Reply:
i was thinking about adding that field, haven't made a decision yet.